app/models/competition.rb

```diff
@@ -96,6 +96,10 @@ class Competition < ApplicationRecord
     restricted: 2,
   }, prefix: true
 
+  NEW_REG_SYSTEM_DEFAULT = :v2
```

Comment:
Well because this is only for distinguishing whether the "new registration system" (as it appears in the competition form) maps to v2 or v3. The old legacy system will always stay as v1